### ISSUE TITLE
Only english support for the 'Coldcard Hardware Wallet Plugin'

### ISSUE THREAD

**Issue by t-a-m-e** [spesmilo/electrum#8676]
There isn't support for multiple languages yet apparently?
What language/s are most needed? Is there are people out there that would like support in a specific language? Maybe there is a logical market that speaks another language natively?
I'd like to work on this, but am unsure of the best place to start.

**Comment by accumulator** [comment]
Indeed, not all user facing strings for the coldcard plugin are translatable. You can have a look at the files in https://github.com/spesmilo/electrum/tree/master/electrum/plugins/coldcard and check if all strings presented to the user are translatable (which means it should be wrapped in a `_()` func, e.g. `_('Translate me')`). 

All strings wrapped in `_()` will appear on https://crowdin.com/project/electrum, where they can be translated to all supported languages.
